_clients/java.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+title: OpenSearch Java client
+nav_order: 65
+---
+
+# Java client
+
+The OpenSearch Java client allows you to interact with your OpenSearch clusters through Java methods and data structures rather than HTTP methods and raw JSON.
+
+For example, you can submit requests to your cluster using objects to create indices, add data to documents, or complete some other operation using the client's built-in methods.
+
+## Setup
+
+To start using the OpenSearch Java client, ensure that you have the following dependency in your project's `pom.xml` file:
+
+```
+<dependency>
+  <groupId>org.opensearch.client</groupId>
+  <artifactId>opensearch-java</artifactId>
+  <version>0.1.0</version>
+</dependency>
+```
+
+You can now start your OpenSearch cluster.
```

Comment:
The following example depends on docker opensearch image that we ship with fixed credentials, We should call out this.